include/qemu/host-utils.h

```diff
@@ -142,6 +143,19 @@ static inline int clz64(uint64_t val)
     return val ? __builtin_clzll(val) : 64;
 }
 
+/**
```

Comment:
nit: extra `*`
Also, the commit subject usually has a subsystem prefix. It's a bit complicated with files like host-utils, but `git blame` suggests the prefix would be `host-utils:`

Reply:
QEMU style shows a single `*`, I don't see why these methods would be different.